src/components/Comments/CreateComments.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const CreateComments = ({
+  handleCommentSubmit,
+  handleInputFieldChange,
+  content,
+  isAuthenticated,
+  contentError,
+  handleClear
+}) => (
+  <section>
+    <div className="flex flex-col w-full m-auto  mb-10 mt-10">
+      <p className="mb-3 font-bold text-2xl">COMMENTS</p>
```

Comment:
this should be a header. Judging from the design of the page, decide what heading level it should be starting from `h1` for the top-level header.

Reply:
Provide an accessible label for the comment form. eg

```html
<form onSubmit={handleCommentSubmit} data-testid="submit-form" arai-label="comment">
```

Reply:
Feedback taken and done!